src/js/views/forms/form/form.scss

```diff
@@ -47,7 +65,7 @@
   border: 1px solid $black-90;
   border-bottom: 0;
   border-radius: 4px 4px 0 0;
-  height: calc(100% - 80px);
+  height: 100%;
```

Comment:
From my manual testing in my local dev environment and what I could see in Cypress, the height of the iframe element maintains the correct height using simply `height: 100%`. 

There may be other side effects of this I'm not aware of in the UI...

Reply:
This is interesting.  The calc I know is a hold-over style from the previous app which supported old-IE.  So it's possibly related.  We should make sure forms are tested on iPad for this release

Reply:
I'll make a note in the Shortcut story for Sofia.